_posts/2015-04-13-Prepare-to-Jam.markdown

```diff
@@ -0,0 +1,96 @@
+</tr>
+<tr>
+<td> Minsk, Belarus </td>
+<td> <a href="https://www.facebook.com/groups/iosby//">CocoaHeadsBy</a></td>
+<td> Aliaksandr Andrashuk, Yana Poteeva at <a href="http://9elements.com">9elements</a> </td>
+</tr>
+<tr>
+<td> London </td>
+<td> <a href="http://www.meetup.com/NSLondon/events/221458856/">NSLondon</a></td>
+<td> Daniel Haight, Raimon Lapuente at <a href="http://facebook.com">Facebook</a> </td>
+</tr>
+<tr>
+<td> Durham, NC </td>
+<td> <a href="http://www.meetup.com/nscoderrtp/events/221366135//">NSCoder </a></td>
+<td> Josh Johnson at <a href="http://americanunderground.com">American Underground</a> </td>
+</tr>
+</tbody>
+</table>
+
+Don't worry if you don't live in a big city, we'll all be online, follow through to find out the details.
+
+<!-- more -->
+
+### [To re-cap](/Test-Jammin):
+
+*TLDR/Elevator Pitch*: If the community is going to move towards Swift, then let’s make sure that our Objective-C is in order by adding tests to established libraries.
+
+We have picked out 200 pods without tests, or with very low test coverage, ordered by popularity. These are assigned randomly to interested contributors. They then will aim to:
+
+* Add tests to the library.
+* Add Continuous Integration.
+* Add Code Coverage support.
+* Submit a Pull Request to the original repo.
+
+We have already contacted all of the library authors letting them know that this is happening and that it would be cool for them participate. Some authors have written issues stating how they would like to have tests set up.
+
+### How it works
+
+We will be focused on trying to improve existing tooling, but this does not mean that we will not be supportive for anyone taking the time to add tests to their own libraries. This is a great time to devote a day or two to improving your own libraries surrounded by people who are in a similar mindset. It will be the perfect time to ask questions in a community. We will be running the chat room based on the <a href="https://www.recurse.com/manual#sub-sec-social-rules">Social Rules</a> of the Recurse Center. We're aiming to have a fun space for people to get started with Testing in.
+
+We will be using IRC to chat, and to interact with our bot. We will be keeping stats inside a web page on [CocoaPods.org](https://cocoapods.org/testjam).
+
+If you are new to IRC, you can connect either via [Limechat](https://itunes.apple.com/gb/app/limechat/id414030210?mt=12) (Mac App Store), a [web-IRC adaptor](https://webchat.freenode.net) or any client of your choice. We will be using the channel [#cocoapods](irc://chat.freenode.net#cocoapods) and the [CocoaPods Test Jam](https://cocoapods.org/testjam) page will have information on how to get started with interacting.
+
+### Useful links
+
+Here's a collection of useful links around the Jam and Testing in general:
+
+ * [CocoaPods Test Jam Stats](https://cocoapods.org/testjam)
+ * [chat.freenode.net#cocoapods](irc://chat.freenode.net#cocoapods)
```

Comment:
Should be a slash between channel and the host. The # should also be URI encoded as it's an unsafe URI character.